src/test/java/racingGame/CarTest.java

```diff
@@ -0,0 +1,40 @@
+package racingGame;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class CarTest {
+    @Test
+    @DisplayName("전략이 true를 반환하면 자동차는 한 칸 전진한다")
+    void move_whenStrategyIsTrue_increasePosition() {
+        Car car = new Car();
+
+        car.move(new AlwaysMoveStrategy());
```

Comment:
```suggestion
        car.move(() -> true);
        car.move(() -> false);
```
람다를 활용해 위와 같이 구현하면 굳이 AlwaysMoveStrategy, NeverMoveStrategy와 같은 클래스를 추가하지 않아도 됨